### PR TITLE
Support for Arrival Sensor KMPCIL tagv1

### PR DESCRIPTION
**Pull request by iharyadi**
@Koenkk,  here is the converter for Arrival Sensor.  This request should have all the lint errors cleaned up.

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -5178,6 +5184,35 @@ const converters = {
             return {state: (msg.data['presentValue']==0) ? 'OFF' : 'ON'};
         },
     },
+    kmpcil_presence_binary_input: {
```
[the remark this answers is not in the document]
Comment (iharyadi) [reply converters/fromZigbee.js:5187]: Hi Koenkk,

I looked at the code.  If I move the code from lib/kmpcil.js to devices/kmpcil.js,  then from fromZigbee.js must include devices/kmpcil.js.  Is that ok for you?   I see the example with xiaomi is that Xiaomi has lib/xiaomi.js that is included in fromZigbee.js

Please let me know what do you think is correct.

Thank you 
Iman

**Comment by iharyadi** [comment]
@Koenkk, Please ignore my last reply.  I think I understand your suggestion now.  I just moved all the codes into device/kmpcil.js and has checked it in

**Comment by Koenkk** [comment]
Added one more comment, could you also make a pr for https://github.com/koenkk/zigbee2mqtt.io, then this can be merged.

**Comment by iharyadi** [comment]
@Koenkk, I just looked at https://github.com/koenkk/zigbee2mqtt.io.  I know that I need to add documentation for my device.  May I have your guidance where should I add the documentation.  There are many files in the source tree.

**Comment by Koenkk** [comment]
Two files need to be added:

- `KMPCIL-tag-001.md` file here: https://github.com/Koenkk/zigbee2mqtt.io/tree/master/docs/devices . On this page write info how this device can be created (a link is also enough)
- `KMPCIL-tag-001.jpg` file here: https://github.com/Koenkk/zigbee2mqtt.io/tree/master/public/images/devices (should be 150x150)

**Comment by iharyadi** [comment]
Thank you for the information.  I will work on the documentation.

**Comment by iharyadi** [comment]
Hi @KoenKK,

In other device,  there is a note in the documentation like below.  My questions for you is that do I need just to filled the information in the note section and check in the file?

---
title: "KMPCIL KMPCIL_RES005 control via MQTT"
description: "Integrate your KMPCIL KMPCIL_RES005 via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
addedAt: 2020-06-29T17:33:31Z
pageClass: device-page
---

**Comment by Koenkk** [comment]
Just filling the note section is enough, the rest will be generated automatically with the next z2m release.

**Comment by iharyadi** [comment]
I am just curious whether there are any further changes needed for this PR.

**Comment by Koenkk** [comment]
Forgot to merge, thanks for pining me!

**Comment by iharyadi** [comment]
Thank you

**Pull request merged**